src/assets/svg/illustration-hero.svg

```diff
@@ -0,0 +1 @@
m-27.752 49.9993 27.752-14.343-1.8366-3.554-27.752 14.344zm28.8337-16.12v-32.1026h-4v32.1026zm-59.50392-32.1026v32.1026h4v-32.1026zm1.081701 33.8796 27.751919 14.343 1.8366-3.553-27.75191-14.344z" fill="#000" fill-opacity=".5" mask="url(#e)"/><path d="m37.748 99.5538 27.752-14.3437 27.7519 14.3437v32.1022l-27.7519 14.344-27.752-14.344z" fill="#ef5aa0"/><path d="m37.2889 100.442 27.7519 14.344.9183-1.777-27.7519-14.3436zm27.2111 13.455v32.103h2v-32.103zm1.4591.889 27.752-14.344-.9183-1.7766-27.752 14.3436zm-.4591-29.5759.9183-1.7768-.9183-.4746-.9183.4746zm-27.752 14.3437-.9183-1.7768-1.0817.5591v1.2177zm55.5039 0h2v-1.2177l-1.0817-.5591zm-27.7519 46.4462-.9183 1.777.9183.474.9183-.474zm27.7519-14.344.9183 1.777 1.0817-.559v-1.218zm-55.5039 0h-2v1.218l1.0817.559zm26.8337-48.2227-27.752 14.3437 1.8366 3.553 27.752-14.3432zm29.5885 14.3437-27.7519-14.3437-1.8366 3.5535 27.7519 14.3432zm-27.7519 50 27.7519-14.344-1.8366-3.553-27.7519 14.343zm28.8336-16.121v-32.1022h-4v32.1022zm-59.5039-32.1022v32.1022h4v-32.1022zm1.0817 33.8792 27.752 14.344 1.8366-3.554-27.752-14.343z" fill="#000" fill-opacity=".5" mask="url(#f)"/><path d="m.745483 36.1207 27.751917-14.3437 27.752 14.3437v32.1026l-27.752 14.3437-27.751917-14.3437z" fill="#f6685c"/><path d="m.286332 37.009 27.751968 14.3437.9183-1.7767-27.75197-14.3437zm27.211068 13.4554v32.1026h2v-32.1026zm1.4592.8883 27.7519-14.3437-.9183-1.7767-27.7519 14.3437zm-.4592-29.5757.9183-1.7767-.9183-.4747-.9183.4747zm-27.751917 14.3437-.918302-1.7767-1.081701.559v1.2177zm55.503917 0h2v-1.2177l-1.0817-.559zm-27.752 46.4463-.9183 1.7767.9183.4746.9183-.4746zm27.752-14.3437.9183 1.7767 1.0817-.5591v-1.2176zm-55.503917 0h-2.000003v1.2176l1.081701.5591zm26.833617-48.223-27.751919 14.3437 1.836609 3.5534 27.75191-14.3437zm29.5886 14.3437-27.752-14.3437-1.8366 3.5534 27.752 14.3437zm-27.752 49.9997 27.752-14.3437-1.8366-3.5535-27.752 14.3437zm28.8337-16.1204v-32.1026h-4v32.1026zm-59.50392-32.1026v32.1026h4v-32.1026zm1.081701 33.8793 27.751919 14.3437 1.8366-3.5535-27.75191-14.3437z" fill="#000" fill-opacity=".5" mask="url(#g)"/><path d="m37.748 57.2651 27.752-14.3437 27.7519 14.3437v32.1025l-27.7519 14.3434-27.752-14.3434z" fill="#6378ba"/><path d="m37.2889 58.1534 27.7519 14.3437.9183-1.7767-27.7519-14.3437zm27.2111 13.4554v32.1022h2v-32.1022zm1.4591.8883 27.752-14.3437-.9183-1.7767-27.752 14.3437zm-.4591-29.5757.9183-1.7768-.9183-.4746-.9183.4746zm-27.752 14.3437-.9183-1.7768-1.0817.5591v1.2177zm55.5039 0h2v-1.2177l-1.0817-.5591zm-27.7519 46.4459-.9183 1.777.9183.475.9183-.475zm27.7519-14.3434.9183 1.7767 1.0817-.559v-1.2177zm-55.5039 0h-2v1.2177l1.0817.559zm26.8337-48.223-27.752 14.3437 1.8366 3.5535 27.752-14.3437zm29.5885 14.3437-27.7519-14.3437-1.8366 3.5535 27.7519 14.3437zm-27.7519 49.9997 27.7519-14.3437-1.8366-3.5534-27.7519 14.3441zm28.8336-16.1204v-32.1025h-4v32.1025zm-59.5039-32.1025v32.1025h4v-32.1025zm1.0817 33.8792 27.752 14.3437 1.8366-3.553-27.752-14.3441z" fill="#000" fill-opacity=".5" mask="url(#h)"/></svg>
```

Comment:
I figured out how to export the illustrations for both the new Landing Page and the Landing Page as SVGs instead of PNGs. (There were some strokes that were getting messed up in the export. Now fixed.)